BCH-Wunder-Project-team-4/Wunder.io
# Styling cards

## Link to ticket:

https://trello.com/c/eQ0Chuwc/128-styling-cards-for-all-events-experttalks-articles

## Link to feature environment:

http://localhost:3000/

## Changes proposed in this PR:

The style for cards in all-events, all-expertTalks, all-articles.  Additionally Hero banner paragraph updated 

<img width="1268" alt="Screenshot 2023-12-14 at 19 13 48" src="https://github.com/BCH-Wunder-Project-team-4/Wunder.io/assets/117649417/41ac74e0-36eb-400b-847e-5aed34e7d9ed">
<img width="1284" alt="Screenshot 2023-12-14 at 19 14 27" src="https://github.com/BCH-Wunder-Project-team-4/Wunder.io/assets/117649417/2fb80a4f-e174-4087-ab9c-15f14ec4d832">
<img width="1189" alt="Screenshot 2023-12-14 at 19 15 20" src="https://github.com/BCH-Wunder-Project-team-4/Wunder.io/assets/117649417/e86ca256-b332-4ad6-a16f-8d2382962771">
<img width="1678" alt="Screenshot 2023-12-14 at 19 22 42" src="https://github.com/BCH-Wunder-Project-team-4/Wunder.io/assets/117649417/c74a726d-053d-436b-9423-7c64667e6a3f">

## How to test:

1. git fetch origin frameParagraph
2. git checkout frameParagraph
3. lando drush cim
4. create listings for  events, articles, experttalks
5. create few events articles expertTalks  if you don't have any
6. check the usability of all-events, all-expertTalks, all-articles.
7. dark mode, colors, lighthouse
